trade-finance-manager-ui/server/controllers/case/cancellation/effective-from-date.controller.ts

```diff
@@ -8,12 +8,23 @@ import { validateEffectiveFromDate } from './validation/validate-effective-from-
 import { canSubmissionTypeBeCancelled } from '../../helpers/deal-cancellation-enabled.helper';
 import api from '../../../api';
 
-export type GetEffectiveFromDateRequest = CustomExpressRequest<{ params: { _id: string } }>;
+export type GetEffectiveFromDateRequest = CustomExpressRequest<{ params: { _id: string }; query: { status?: string } }>;
 export type PostEffectiveFromDateRequest = CustomExpressRequest<{
   params: { _id: string };
+  query: { status?: string };
   reqBody: { 'effective-from-date-day': string; 'effective-from-date-month': string; 'effective-from-date-year': string };
 }>;
 
+/**
+ * Gets the previous page URL to pass into the back link
+ *
+ * @param dealId - The deal ID
+ * @param status - 'change' if user comes from the check answers page, otherwise undefined
```

Comment:
```suggestion
 * @param status - 'change' if user comes from the check answers page, otherwise 'cancellation - bank request date' 
```

Reply:
This is the query param in the URL - so it's correct as is! (`?change` is at the end of the URL if the user has arrived at the page by clicking 'change' on the review page, otherwise there's nothing extra at the end of the URL)

Reply:
Sorry, was reading this as the return type 🤦